### PR TITLE
Import hdf5plugin for JLS decompression support if available

### PR DESCRIPTION
**Pull request by pnuu**
This PR brings support to decompression of MTG/FCI L1 files that have been compressed with JLS compression by trying to import `hdf5plugin` library.

Can't think of any unit tests for this, but I did run actual data through and decompression works after this minor change.

 - [ ] Tests added  
 - [ ] Tests passed 
 - [x] Passes ``flake8 trollflow2`` 
 - [x] Fully documented 
 - [ ] Add your name to `AUTHORS.md` if not there already

### CODE REVIEW

**Review by gerritholl — DISMISSED** [review]
Looks good to me

File: doc/source/index.rst [hunk 1]
```diff
@@ -15,6 +15,25 @@ Trollflow2 is an operational generation chain runner for Satpy.
...
+For support of specific satellite data, see `Satpy documentation <https://satpy.readthedocs.io/en/stable/>`_ for the
+necessary packages that need to be installed.
+
+The Trollflow2 package isn't available via Conda, so it needs to be installed with ``pip``::
+
+  pip install trollflow2
+
+For decompression of MTG/FCI L1 files that have been compressed with CharLS compression ``hdf5plugin`` is needed::
+
+  mamba install hdf5plugin
```
Comment (mraspaud) [inline doc/source/index.rst:34]: This is present in the satpy doc, do we want to repeat it here?

**Review by mraspaud — COMMENTED** [review]
Nice, just a few comments.

File: doc/source/index.rst [hunk 2]
```diff
@@ -15,6 +15,25 @@ Trollflow2 is an operational generation chain runner for Satpy.
...
 See the example playlist (``pl.yaml``) for inspiration.
 
+Installation
+------------
+
+Installing dependencies is simplest by using `Mambaforge <https://github.com/conda-forge/miniforge#mambaforge>`_::
+
+  mamba create -n trollflow2 satpy pyresample pyyaml dpath trollsift pyorbital
+
+For support of specific satellite data, see `Satpy documentation <https://satpy.readthedocs.io/en/stable/>`_ for the
```
[the remark this answers is not in the document]
Comment (pnuu) [reply doc/source/index.rst:25]: Actually the only place where the requirements are shown are in [`setup.py`](https://github.com/pytroll/satpy/blob/main/setup.py#L43-L85) and there's pretty much nothing in the Satpy documentation.

File: trollflow2/plugins/__init__.py [hunk 3]
```diff
@@ -28,6 +28,10 @@
 from tempfile import NamedTemporaryFile
 from urllib.parse import urlunsplit
 
+try:
+    import hdf5plugin  # noqa
+except ImportError:
+    pass
```
[the remark this answers is not in the document]
Comment (pnuu) [reply trollflow2/plugins/__init__.py:34]: Don't know, never knew such existed..

File: doc/source/index.rst (hunk 1 above)
Comment (pnuu) [reply doc/source/index.rst:34]: Sure. I'll actually remove the whole installation chapter and create an issue about better documentation if it doesn't already exist.

File: doc/source/index.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (pnuu) [reply doc/source/index.rst]: https://github.com/pytroll/trollflow2/issues/149

File: trollflow2/plugins/__init__.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (pnuu) [reply trollflow2/plugins/__init__.py]: Done.

File: doc/source/index.rst (hunk 1 above)
Comment (pnuu) [reply doc/source/index.rst]: See https://github.com/pytroll/trollflow2/issues/149

**Review by mraspaud — APPROVED** [review]
LGTM

**Pull request merged**